hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -140,8 +142,18 @@ public static void setBucketFSOptimized(boolean enabledFSO) {
    * @return OMClientRequest
    * @throws IOException
    */
-  public static OMClientRequest createClientRequest(OMRequest omRequest) {
+  public static OMClientRequest createClientRequest(OMRequest omRequest,
+      OzoneManager ozoneManager) throws IOException {
```

Comment:
Please remove `throws IOException` from the method signature, which is not required as we are handling exception using the BucketLayout.LEGACY, right?

Reply:
Done